yal/tests/test_main.py

```diff
@@ -256,8 +257,24 @@ async def test_callback_check_response_to_handler(tester: AppTester):
 @pytest.mark.asyncio
 async def test_aaq_timeout_response_to_handler(tester: AppTester, rapidpro_mock):
     await tester.user_input(session=Message.SESSION_EVENT.NEW, content="no")
+    tester.assert_state("state_no_question_not_answered")
     tester.assert_num_messages(1)
-    tester.assert_message("TODO: Handle question not answered")
+    message = "\n".join(
+        [
+            "*I'm sorry I couldn't find what you were looking for this time.* ",
+            "",
+            "Please tell me what you're looking for again. "
+            "I'll try make sure I have the right information "
+            "for you next time.",
+            "",
+            "_Just type and send your question again, now._" "",
+            "-----",
+            "*Or reply:*",
+            BACK_TO_MAIN,
+            GET_HELP,
+        ]
+    )
+    tester.assert_message(message)
```

Comment:
After the changes you'll also need to add `aaq_mock` to this test as well (and the definition for it might not exist in this file yet 😬 ) but you can leave out the tests to check the request that went through